rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/InfrastructureManager.java

```diff
@@ -592,6 +618,14 @@ protected final boolean checkNodeIsAcquiredAndDo(String nodeName, Runnable toRun
 		}
 	}
 
+	protected final boolean checkAllNodesAreAcquiredAndDo(List<String> nodeNames, Runnable toRunWhenOK, Runnable toRunWhenKO) {
+		boolean answer = true;
+		for (String nodeName : nodeNames) {
+			answer = answer && checkNodeIsAcquiredAndDo(nodeName, toRunWhenOK, toRunWhenKO);
```

Comment:
Not in that case, because of the actions which must be taken when the node is not acquired